app/models/annotation.rb

```diff
@@ -51,9 +51,7 @@ def get_data(include_creator: false)
       criterion_id: annotation_text.annotation_category&.flexible_criterion&.id
     }
 
-    if include_creator
-      data[:creator] = creator.display_name
-    end
+    data[:creator] = include_creator && creator ? creator.display_name : I18n.t('users.deleted')
```

Comment:
Don't merge the `include_create` check with the `creator` check; keep the overall `if` statement the way it was before, but then inside use just `creator ? ...` This ensures that the `:creator` key is not included at all when `include_creator` is false, which is the existing (and intended) behaviour.

Also use `creator.present?` instead of just `creator` to be more explicit.